root/edit/components/EditList.js

```diff
@@ -98,8 +98,14 @@ const EditList = ({
 
       {edits.length ? (
         <div id="edits">
-          <PaginatedResults guessSearch={mightBeMoreEdits} pager={pager}>
-            <form action="/edit/enter_votes" method="post">
+          <form action="/edit/enter_votes" method="post">
+            {$c.user ? (
+              <FormSubmit
+                className="float-right"
```

Comment:
It aligns the button to the right but it is missing vertical margins. Would it be better to use the same `<div className="align-right row no-label">` as for the submission button at the bottom?

Reply:
Maybe I'm misunderstanding, but I believe that that results in the button being in its own row, which wastes a bit of vertical space:

![image](https://github.com/metabrainz/musicbrainz-server/assets/40385/b731860a-8960-4dc2-9e66-32f437e0c8f9)

Or I guess it could go under the paginator:

![image](https://github.com/metabrainz/musicbrainz-server/assets/40385/0d9850f9-4cfc-46b3-aeca-e6be2648dd63)

Let me know if you prefer either of those.

Reply:
> that results in the button being in its own row, which wastes a bit of vertical space

Yes, I didn’t notice this issue (waste of space) but it is how it is currently displayed, right?

The margin issue I mentioned is occurring on non-paginated lists of edits.

Reply:
To clarify, I don’t think that having separate rows is a major issue. But if you intend to address it too (on top of MBS-5987), please make it a separate commit and changes it for both buttons (at the top and at the bottom).

Reply:
Got it; thanks for the explanation. I've amended the commit to place the button on its own row below the paginator.